x/collection/keeper/msg_server_test.go

```diff
@@ -198,6 +194,36 @@ func helperBuildEventSent(contractID string, from, to, operator sdk.AccAddress,
 	}
 }
 
+func (s *KeeperTestSuite) TestHelperBuildEventSent() {
+	testCases := map[string]struct {
+		expectedEvent sdk.Event
+		contractID    string
+		from          sdk.AccAddress
+		to            sdk.AccAddress
+		operator      sdk.AccAddress
+		amount        collection.Coins
+	}{
+		"should keep event compatibility": {
+			expectedEvent: sdk.Event{Type: "lbm.collection.v1.EventSent", Attributes: []abci.EventAttribute{{Key: []uint8{0x61, 0x6d, 0x6f, 0x75, 0x6e, 0x74}, Value: []uint8{0x5b, 0x7b, 0x22, 0x74, 0x6f, 0x6b, 0x65, 0x6e, 0x5f, 0x69, 0x64, 0x22, 0x3a, 0x22, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x31, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x22, 0x2c, 0x22, 0x61, 0x6d, 0x6f, 0x75, 0x6e, 0x74, 0x22, 0x3a, 0x22, 0x31, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x22, 0x7d, 0x5d}, Index: false}, {Key: []uint8{0x63, 0x6f, 0x6e, 0x74, 0x72, 0x61, 0x63, 0x74, 0x5f, 0x69, 0x64}, Value: []uint8{0x22, 0x39, 0x62, 0x65, 0x31, 0x37, 0x31, 0x36, 0x35, 0x22}, Index: false}, {Key: []uint8{0x66, 0x72, 0x6f, 0x6d}, Value: []uint8{0x22, 0x6c, 0x69, 0x6e, 0x6b, 0x31, 0x76, 0x39, 0x6a, 0x78, 0x67, 0x75, 0x6e, 0x39, 0x77, 0x64, 0x65, 0x6e, 0x71, 0x61, 0x32, 0x78, 0x7a, 0x66, 0x78, 0x22}, Index: false}, {Key: []uint8{0x6f, 0x70, 0x65, 0x72, 0x61, 0x74, 0x6f, 0x72}, Value: []uint8{0x22, 0x6c, 0x69, 0x6e, 0x6b, 0x31, 0x76, 0x39, 0x6a, 0x78, 0x67, 0x75, 0x6e, 0x39, 0x77, 0x64, 0x65, 0x6e, 0x71, 0x61, 0x32, 0x78, 0x7a, 0x66, 0x78, 0x22}, Index: false}, {Key: []uint8{0x74, 0x6f}, Value: []byte{0x22, 0x6c, 0x69, 0x6e, 0x6b, 0x31, 0x76, 0x39, 0x6a, 0x78, 0x67, 0x75, 0x6e, 0x39, 0x77, 0x64, 0x65, 0x6e, 0x79, 0x6a, 0x71, 0x79, 0x79, 0x78, 0x75, 0x22}, Index: false}}},
+			contractID:    s.contractID,
+			from:          s.vendor,
+			to:            s.customer,
+			operator:      s.vendor,
+			amount:        collection.NewCoins(collection.NewFTCoin(s.ftClassID, s.balance)),
+		},
+	}
+	if !s.deterministic {
```

Comment:
Or, you may create your own literals to remove this conditional.

Reply:
I tried but I couldn't.
I need a different literal for `s.deterministic` because of the difference of the addresses per `s.deterministic`.
Do you have any idea that I could remove this conditional?

Reply:
You should create new addresses in this test. Don't try to use the addresses made by the suite.

Reply:
Okay, I may add some literal input for the event that can be used regardless of changes.

Reply:
Okay, I may add some literal input for the event that can be used regardless of changes.